default/python/AI/ShipDesignAI.py

```diff
@@ -907,6 +912,7 @@ def update_stats(self, ignore_species=False):
                 else:
                     self.design_stats.fighter_capacity += self._calculate_hangar_capacity(part)
                     self.design_stats.fighter_damage = self._calculate_hangar_damage(part)
+                    self.design_stats.fighter_launch_bay_rate += self._calculate_fighter_launch_bay_rate(part)
```

Comment:
This seems wrong: You increase the launch bay rate whenever you add another hangar. Should probably just read = rather than +=.